### PR TITLE
stick to mock < 4.0 when using Python 2

### PR DESCRIPTION
**Pull request by boegel**

### CODE REVIEW

File: lib/vsc/install/shared_setup.py [hunk 1]
```diff
@@ -1564,7 +1564,9 @@ def action_target(package, *args, **kwargs):
         # but vsc-install relies on the setuptools' behaviour of ignoring failing dependency installations and
         # just continuing with the next entry in dependency_links
         'setuptools < %s' % MAX_SETUPTOOLS_VERSION,
-        'mock',
+        # mock 4.0 only supports Python 3+
+        "mock<4.0;python_version<'3.0'",
```
Comment (wdpypere) [inline lib/vsc/install/shared_setup.py]: put spaces around operators for rpms

Comment (boegel) [reply lib/vsc/install/shared_setup.py]: Hmm, not even sure the `;` will be digested well by the RPM stuff...

Comment (boegel) [reply lib/vsc/install/shared_setup.py]: fixed in 8ee66c8, verified with `clusterbuildrpm` using my branch (& `--queryonly`)

**Comment by wdpypere** [comment]
much cleaner

**Pull request merged**